### PR TITLE
Wrap MariaDAO connection and Colyseus binding

### PR DESCRIPTION
**Pull request by Saphs**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -12,80 +12,140 @@ import backendConfig from "./configs/backend-config.json";
...
+        const now = new Date(Date.now()).toLocaleTimeString();
+        const queries = JSON.stringify(req.query);
+        const params = JSON.stringify(req.params);
+        const body = JSON.stringify(req.body);
+        console.log(`[${now}][HTTP] Got: ${req.method} at ${req.originalUrl} with query: ${queries} params: ${params} body ${body}`);
+        next();
+    });
+
+    console.log("Attaching Express routes for:\n\t/colyseus\n\t/api\n\t/");
+    express_app.use('/colyseus', monitor());
```
Comment (manuelstolze) [inline app.ts:81]: pls use app for express variable

File: app.ts [hunk 2]
```diff
@@ -12,80 +12,140 @@ import backendConfig from "./configs/backend-config.json";
...
+            choices: ['dev', 'prod'],
+            demandOption: true,
+            description: 'app environment'
+        }
+    }).argv;
+
+    let config;
+    if (argv.env === 'prod') {
+        config = backendConfig;
+        config.env = 'prod'
```
Comment (manuelstolze) [inline app.ts:34]: Kann man das `prod`  nicht in die backendConfig packen? Das selbe bei der Dev Variante?

**Review by manuelstolze — APPROVED** [review]

**Comment by DanielLiebler** [comment]
This is NOT WORKING! Testing should be crucial. The route for logging in, /matchmake/game , is directed to the route for / .
This makes the creation of a game impossible!
I suspect the workaround in [app.ts:144-172](https://github.com/Saphs/BrettSpiel_Backend/blob/ea47ecbad17424e4d3c957bb957499eb50b5e828/app.ts#L144-L172) to be the main cause.
This is of importance because it hinders #15 from being tested and merged!

**Comment by DanielLiebler** [comment]
@Saphs

**Comment by DanielLiebler** [comment]
Fixed in commit 6a0a7ec1de5d39130e52cf87906bc2d9c68b1a9c

**Pull request merged**